pkg/strategy/drift/strategy.go

```diff
@@ -199,11 +199,19 @@ func (s *Strategy) ClosePosition(ctx context.Context, percentage fixedpoint.Valu
 		if s.Market.IsDustQuantity(order.Quantity, price) {
 			return nil
 		}
-		_, err := s.GeneralOrderExecutor.SubmitOrders(ctx, *order)
+		createdOrders, err := s.GeneralOrderExecutor.FastSubmitOrders(ctx, *order)
 		if err != nil {
 			order.Quantity = order.Quantity.Mul(fixedpoint.One.Sub(Delta))
 			continue
 		}
+
+		if createdOrders != nil {
```

Comment:
lint issue, it's a slice, so you don't need to check nil

Reply:
changed the interface.